src/he/conn.h

```diff
@@ -485,4 +485,28 @@ he_connection_protocol_t he_conn_get_current_protocol(he_conn_t *conn);
  */
 const char *he_conn_get_curve_name(he_conn_t *conn);
 
+/**
+ * @brief Tell Helium to start a PMTU discovery
+ * @param conn A pointer to a valid connection
+ * @return HE_SUCCESS A PMTU discovery is started
+ * @return HE_ERR_INVALID_STATE if a PMTU discovery is already in-progress
```

Comment:
If the PMTUD callbacks are not set, should we return an error here too?

Reply:
Actually, I see you created `HE_ERR_PMTUD_CALLBACKS_NOT_SET`  in https://github.com/expressvpn/lightway-core/pull/127/commits/af7e922c0003daef97d860c974314704834e6035  so we can simply document that.

I also don't see any place where `HE_ERR_INVALID_STATE` is returned; perhaps this comment should be updated to remove that reference too?

Reply:
Fixed